### DISCUSSION TITLE
There is no role named `Admin`.

### DISCUSSION THREAD

**Discussion by badrujr** [spatie/laravel-permission#2306] [Q&A]
Hello am trying to give role to my user but i receive this error, "There is no role named `Admin`."
#Laravel#Spatie#role

**Comment by erikn69** [comment]
Check the default guard and role guard_name

**Reply by badrujr** [reply]
okay let me look into it

**Reply by parallels999** [reply]
> Greetings, I already solve it thank you

So, mark this as answered, this way you avoid wasting other people's time in the future

**Comment by sethingham** [comment]
Are you still having this issue? Can you provide any more detail? What version are you running of this, laravel and php?

**Comment by badrujr** [comment]
Greetings, I already solve it thank you

**Reply by titanen** [reply]
How did u solve it? I have the same issue.

**Reply by parallels999** [reply]
https://github.com/spatie/laravel-permission/discussions/2306#discussioncomment-4706922
> Check the default guard and role guard_name

**Reply by titanen** [reply]
Do I need to add the default guard namn to guard_name for all roles and permission?
